client/components/edit/datastream/DatastreamLicenseContent.test.tsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import { describe, beforeEach, expect, it, jest } from "@jest/globals";
+import { mount } from "enzyme";
+import { act } from "react-dom/test-utils";
+import toJson from "enzyme-to-json";
+import DatastreamLicenseContent from "./DatastreamLicenseContent";
+
+const mockUseEditorContext = jest.fn();
+jest.mock("../../../context/EditorContext", () => ({
+    useEditorContext: () => {
+        return mockUseEditorContext();
+    },
+}));
+const mockUseDatastreamOperation = jest.fn();
+jest.mock("../../../hooks/useDatastreamOperation", () => () => {
+    return mockUseDatastreamOperation();
+});
+
+describe("DatastreamLicenseContent", () => {
+    let editorValues;
+    let datastreamOperationValues;
+    beforeEach(() => {
+        editorValues = {
+            action: {
+                toggleDatastreamModal: jest.fn(),
+            },
+        };
+        datastreamOperationValues = {
+            uploadLicense: jest.fn(),
+            getLicenseKey: jest.fn(),
+        };
+        mockUseEditorContext.mockReturnValue(editorValues);
+        mockUseDatastreamOperation.mockReturnValue(datastreamOperationValues);
+    });
+
+    it("renders", () => {
+        const wrapper = mount(<DatastreamLicenseContent />);
+        expect(toJson(wrapper)).toMatchSnapshot();
```

Comment:
Would it make sense to mock some of the MUI components in this test to make the snapshot more concise/readable? As it is, the snapshot is incredibly huge. I realize that snapshots aren't necessarily intended to be human-readable, but it still might be worth making it a little more review-friendly.